kidsgramming-fe/src/alumno/views/RegisterForm.jsx

```diff
@@ -0,0 +1,83 @@
+import Grid from '@mui/material/Grid'
+import TextField from '@mui/material/TextField'
+import Alert from '@mui/material/Alert'
+import Button from '@mui/material/Button'
+import Typography from '@mui/material/Typography'
+import Link from '@mui/material/Link'
+import PropTypes from 'prop-types'
+import { Link as RouterLink } from 'react-router-dom'
+import { useMemo } from 'react'
+import { useSelector } from 'react-redux'
+
+import { AuthFormLayout } from '../layout/AuthFormLayout'
+import { authenticationStatuses } from '../../store/auth/authSlice'
+
+export const RegisterForm = ({ authForm }) => {
+  const { status, errorMessage } = useSelector(state => state.auth)
+  const isAuthenticating = useMemo(() => status === authenticationStatuses.authenticating, [status])
```

Comment:
There is no need to verify authentication since the parent is already authenticated at this point